tests/unit/backend/recipe/algorithm/test_FitMultiplePeaksAlgorithm.py

```diff
@@ -0,0 +1,51 @@
+import os
+import unittest.mock as mock
+
+with mock.patch.dict(
+    "sys.modules",
+    {
+        "snapred.backend.log": mock.Mock(),
+        "snapred.backend.log.logger": mock.Mock(),
+    },
+):
+    from mantid.simpleapi import LoadNexusProcessed
+    from snapred.backend.dao.calibration.Calibration import Calibration
+    from snapred.backend.dao.CrystallographicInfo import CrystallographicInfo
+    from snapred.backend.dao.FitMultiplePeaksIngredients import FitMultiplePeaksIngredients
+    from snapred.backend.recipe.algorithm.FitMultiplePeaksAlgorithm import (
+        FitMultiplePeaksAlgorithm,  # noqa: E402
+    )
+    from snapred.meta.Config import Resource
+
+    def test_init():
+        """Test ability to initialize purge overlapping peaks algo"""
+        instrumentState = Calibration.parse_raw(
+            Resource.read("/inputs/purge_peaks/input_parameters.json")
```

Comment:
update resources? or explain

Reply:
Fit Multiple Peaks calls Purge Overlapping Peaks using that information so the resources used for testing are the same.